zokrates_core/src/flatten/mod.rs

```diff
@@ -2147,6 +2202,36 @@ impl<'ast, T: Field> Flattener<'ast, T> {
         }
     }
 
+    fn flatten_assertion(
```

Comment:
Would this be clearer is named `flatten_equality` ?

Reply:
yes this is more clear!